### DISCUSSION TITLE
RTK with PocketSDR

### DISCUSSION THREAD

**Discussion by fabio-f68** [tomojitakasu/PocketSDR#18] [Ideas]
It's possible to integrate rtknavi kinematic mode in pocket SDR ?

**Comment by tomojitakasu** [comment]
Basically yes but the quality of the carrier-phase has not yet been tested.

The problems might be higher noise level of phase and inperfect half-cycle ambiguity resolution.

I expect that AR would be harder compared to commercial RTK receivers. More experience and additional effort might  be necessary to obtain practical performance level for RTK.
